tokio/src/sync/mpsc/bounded.rs

```diff
@@ -477,3 +507,32 @@ impl<T> Sender<T> {
         }
     }
 }
+
+cfg_block_on! {
+    impl<T> Sender<T> {
```

Comment:
In this specific case I prefer to manually write the `#[cfg(...)]` instead of using our `cfg_*` macros. This is because having separate impl blocks clutters the documentation for the channel.

Can you move it into the existing impl block between `send_timeout` and `poll_ready`? You can see on the `send_timeout` method how to do it manually without the `cfg_*` macro.

```rust
#[cfg(feature = "time")]
#[cfg_attr(docsrs, doc(cfg(feature = "time")))]
pub async fn send_timeout(
    &mut self,
    value: T,
    timeout: Duration,
) -> Result<(), SendTimeoutError<T>> {
```

Reply:
Will do, I should get to it in the next 2-3 hours :+1: